codebuild/cd/test-fips-branch.sh

```diff
@@ -9,12 +9,12 @@ cd ./crt/aws-lc || exit 1
 # Get the current commit hash
 current_commit=$(git rev-parse HEAD)
 
-# Check if the current commit is from the "fips-2022-11-02" branch
-if git merge-base --is-ancestor "$current_commit" "origin/fips-2022-11-02"; then
-    echo "Current aws-lc commit is from the 'fips-2022-11-02' branch"
+# Check if the current commit is from the "AWS-LC-FIPS-3.0.0" branch
```

Comment:
Is AWS-LC-FIPS-3.0.0 approved yet?

Reply:
We link the NIST Certificate from the [README.md](https://github.com/awslabs/aws-crt-java?tab=readme-ov-file#fips-compliance)

This should be updated. Do we have a link to the new certificate?

Reply:
Updated Readme